src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black,
+    Bronze,
+    Gold,
+    Silver,
+    White
```

Comment:
we should write enum values in uppercase
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names